microsoftgraph/microsoft-graph-explorer-v2
# Feat/query response

## Overview

1.  Adds a QueryResponse Component
2.  Integrates Monaco Editor
3.  Eject Create React App, leading to the automatic addition of files in the `config` and `scripts` folders

### Demo

![image](https://user-images.githubusercontent.com/12011447/53766670-823cd080-3ee4-11e9-87cb-e6790509a6a3.png)

![image](https://user-images.githubusercontent.com/12011447/53770662-0b5a0480-3ef1-11e9-84ee-d51fcf688431.png)

### Notes
We had to eject Create React App so that we can access the webpack config file.
Making the editor `read only` prevents formatting of the `json` content. 

This pull request doesn't add any React Component tests. This is because of the transient failure of the jest runtime when transforming `editor.api.js`. The fixes suggested [here](https://github.com/facebook/jest/issues/6933) works for some cases and fails on others. They did not work for our use case.

## Testing Instructions

* Pull changes
* Run a query
* Observe the response body being populated with formatted json content.


## Fixes
#9 
#8 
#2
